integration/normal.t.pl

```diff
@@ -34,13 +34,25 @@
     $setattr = "setextattr -h user encfs hello";
     $getattr = "getextattr -h user encfs";
 }
+
 # Do we support xattr ?
 my $have_xattr = 1;
 if(system("./build/encfs --verbose --version 2>&1 | grep -q HAVE_XATTR") != 0)
 {
     $have_xattr = 0;
 }
 
+# Did we ask, or are we simply able to run "sudo" tests ?
+my $sudo_cmd;
+if (! $<)
```

Comment:
should this be `$>` to check effective user id rather than real user id?

Please make the test condition explicit, eg `if ( $> != 0 ) {`

Reply:
Updated 👍